### PR TITLE
helm mapkubeapis plugin - also in offline packages

### PR DESCRIPTION
**Pull request by jschaul**
See also https://github.com/wireapp/wire-server/pull/3019 and https://github.com/zinfra/cailleach/pull/1502

### CODE REVIEW

File: nix/overlay.nix [hunk 1]
```diff
@@ -1,4 +1,7 @@
-self: super: {
+self:
+let helm-mapkubeapis = self.callPackage ./pkgs/helm-mapkubeapis.nix { };
```
Comment (supersven) [inline nix/overlay.nix:2]: I was a bit surprised seeing this expression here.

Have you tried to move it closer to its usage? E.g. to line 16?

**Review by supersven — APPROVED** [review]
Overall, LGTM :+1: 

I've left one comment. Solving it might make the code easier to digest.

File: nix/overlay.nix (hunk 1 above)
Comment (jschaul) [reply nix/overlay.nix:2]: I did. But then it gave me "variable helm-mapkubeapis unknown" or something similar. When I moved it up here it worked.

Comment (supersven) [reply nix/overlay.nix:2]: It's not that bad. So, never mind :shrug: :smile:

**Pull request merged**